scripts/e2e/setup-and-run-e2e.sh

```diff
@@ -4,11 +4,13 @@ export LOCAL_REGISTRY_PORT=12345
 export LOCAL_REGISTRY=dev.localhost
 export DOCKER_REGISTRY=${LOCAL_REGISTRY}:${LOCAL_REGISTRY_PORT}
 export VERSION=test-local
+export MLP_CHART_VERSION=0.4.18
+export MERLIN_CHART_VERSION=0.10.16
 
 
 # Create k3d cluster and managed registry
 k3d registry create $LOCAL_REGISTRY --port $LOCAL_REGISTRY_PORT
-k3d cluster create $K3D_CLUSTER --image rancher/k3s:v1.22.15-k3s1 --k3s-arg '--no-deploy=traefik,metrics-server@server:*' --port 80:80@loadbalancer
+k3d cluster create $K3D_CLUSTER --image rancher/k3s:v1.25.9-k3s1 --k3s-arg '--disable=traefik,metrics-server@server:*' --port 80:80@loadbalancer
```

Comment:
Could you capture the motivation for this change in the PR description, for future reference?

Reply:
I've updated PR description 👍🏻